src/main/kotlin/com/tylerthrailkill/helpers/prettyprint/PrettyPrint.kt

```diff
@@ -93,6 +93,7 @@ private class PrettyPrinter(val tabSize: Int, val writeTo: Appendable, val wrapp
             obj is String -> ppString(obj, collectionElementPad)
             obj is Enum<*> -> ppEnum(obj)
             obj.isAtomic() -> ppAtomic(obj)
+            obj is Array<*> -> ppArray(obj, collectionElementPad)
```

Comment:
Actually you could just call `ppIterable` here, like so `obj is Array<*> -> ppIterable(obj.toList, collectionElementPad)` and add the comment to the end.